features/step_definitions/appointment_steps.rb.rb

```diff
@@ -1,3 +1,21 @@
+Given(/^we import booking bug appointment data between "([^"]*)" and "([^"]*)"$/) do |begin_date, end_date|
+  # filter import on date as date dimension is populated from seeds
+  begin_date = Date.parse(begin_date)
+  end_date = Date.parse(end_date)
+
+  original_find_by = Dimensions::Date.method(:find_by!)
+
+  allow(Dimensions::Date).to receive(:find_by!) do |query|
```

Comment:
I'm not sure we should be stubbing objects in an acceptance test. We want to exercise as much of the system as possible.